control.lua

```diff
@@ -161,10 +161,12 @@ local function create_entities(proxy, tags, rail_poss)
   local inserter_name =
     "rail" .. type .. (allowed_items_setting == "any" and "-universal" or "") .. "-inserter"
   for i=1,num_inserters do
+    -- alternate direction to support half-size wagons stick out both sides of the (un)loader
+    local inserter_direction = direction + (i-1) * 4
```

Comment:
Suggest adding a `% 8` in case `num_inserters` increases beyond 2 at some point.